deploy/oasis-v2/skillopt/register_oasis_env.py

```
@@ -0,0 +1,28 @@
+    except ImportError:
+        pass
+\n"""
+
+
+def patch(path: Path) -> None:
+    text = path.read_text(encoding="utf-8")
+    if 'skillopt.envs.oasis.adapter' in text:
+        return
+    anchor = 'def get_adapter(cfg: dict):'
+    if anchor not in text:
+        raise RuntimeError(f"Point d’enregistrement SkillOpt absent : {path}")
```

Ruff (0.16.2)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)